net-mail/vacation/vacation-1.2.7.0-r1.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils toolchain-funcs
+
+DESCRIPTION="automatic mail answering program"
+HOMEPAGE="http://vacation.sourceforge.net/"
+SRC_URI="mirror://sourceforge/vacation/${P}.tar.gz"
+LICENSE="GPL-2"
+KEYWORDS="~alpha ~amd64 ~x86"
+SLOT="0"
+
+RDEPEND="virtual/mta
+	sys-libs/gdbm"
+DEPEND="${RDEPEND}
+	!mail-mta/sendmail"
+
+src_unpack() {
+	unpack ${A}
+	cd "${S}"
+}
+
+src_prepare() {
+	eapply_user
+
+	sed -i -e "s:install -s -m:install -m:" Makefile || die "sed failed!"
+	sed -i -e "s:-Xlinker:${LDFLAGS} -Xlinker:" Makefile || die "sed failed!"
+}
+
+src_compile () {
+	emake CC=$(tc-getCC) ARCH=$(tc-arch-kernel) CFLAGS="${CFLAGS} -DMAIN" || die "emake failed."
```

Comment:
emake has it's own internal die, no need here.

Reply:
Ok, will do.
Thank you